airflow/providers/databricks/operators/databricks.py

```diff
@@ -1039,6 +1123,17 @@ def launch_notebook_job(self) -> int:
         self.log.info("Check the job run in Databricks: %s", url)
         return self.databricks_run_id
 
+    def _handle_terminal_run_state(self, run_state):
+        if run_state.life_cycle_state != "TERMINATED":
```

Comment:
Should we use `RunState.is_terminal`?

Reply:
This method `_handle_terminal_run_state` is already called `Runstate.is_terminal`. `Runstate.is_terminal` is reached & decided as so when the `life_cycle_state` is either of ("TERMINATED", "SKIPPED", "INTERNAL_ERROR"). And here, we want to raise an error if it not equal to `TERMINATED`